exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -224,10 +224,16 @@ so you may run "pnpm -w run ${scriptName}"`,
     })
   }
   const concurrency = opts.workspaceConcurrency ?? 4
+
+  const extraEnv = {
+    ...opts.extraEnv,
+    ...(opts.nodeOptions ? { NODE_OPTIONS: opts.nodeOptions } : {}),
```

Comment:
What if `NODE_OPTIONS` is already defined in the environment?

Reply:
It overrides it, as in npm
```powershell
PS D:\code\support-node-options> $env:NODE_OPTIONS = "--experimental-vm-modules"
PS D:\code\support-node-options> node -e "console.log(process.env.NODE_OPTIONS)"
--experimental-vm-modules
PS D:\code\support-node-options> npm run heap

> support-node-options@1.0.0 heap
> node -e "console.log(process.env.NODE_OPTIONS)"

--max-old-space-size=1024
PS D:\code\support-node-options> pd heap

> support-node-options@1.0.0 heap D:\code\support-node-options
> node -e "console.log(process.env.NODE_OPTIONS)"

--max-old-space-size=1024
```